src/core.jl

```diff
@@ -48,6 +61,9 @@ Return the total number of parameters of the layer `l`.
 function parameterlength(l::AbstractExplicitLayer)
     return parameterlength(initialparameters(Random.default_rng(), l))
 end
+function parameterlength(::AbstractExplicitLayer{hasstate, false}) where {hasstate}
```

Comment:
Ordering is incorrect

Reply:
Fixed in [37f4851](https://github.com/avik-pal/Lux.jl/pull/84/commits/37f48519a731897b9de5771ccb714ff0eea2e013)